pkg-cmp-between.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Bash script that checks if the given RPM version is newer/equal to EVR1 and older/equal to EVR2
+# Exits 0 if RPM is newer/equal to EVR1 and older/equal to EVR2, 1 otherwise
+
+USAGE="Usage: pkg-cmp-between.sh <RPM> <EVR1> <EVR2>"
+
+RPM=$1
+EVR1=$2
+EVR2=$3
+
+if [[ $# -ne 3 ]]; then
+    echo "Expected 3 args, got $#"
+    echo "$USAGE"
+    exit
```

Comment:
```suggestion
    exit 2
```
The standard exit code for incorrect usage is 2.